### ISSUE TITLE
Indentation error on returning parent wrapped boolean expression

### ISSUE THREAD

**Issue by sarchy** [checkstyle/checkstyle#13547]
Sometimes when I come across lengthy boolean checks I want to put them in a separate method. When wrapping I looks nicer to wrap them in parens and put every top level clause on a separate line, closing the the expressions with an `)` at the same indentation level as the `(`.

Checkstyle seems to support this just fine when assigning to a variable. But when combined with a `return` it insists that the `)` should be at the same indentation level as the other statements (which is ugly and inconsistent with regular assignment).

```java
public class Example {
	public static boolean isBlankOne(String s) {
		return (
			s != null &&
			s.isBlank()
		);
	}
	public static boolean isBlankTwo(String s) {
		boolean result = (
			s != null &&
			s.isBlank()
		);
		return result;
	}
}
```

```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN" "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">

	<property name="tabWidth" value="4"/>
	<module name="TreeWalker">
		<module name="Indentation" />
	</module>

</module>
```

```cmd
$ java -jar ~/Downloads/checkstyle-10.12.2-all.jar -c checkstyl.xml **/*.java | sed -e "s:${PWD}:.:"
Starting audit...
[ERROR] ./main/Example.java:7:9: 'method def' child has incorrect indentation level 8, expected level should be 12. [Indentation]
Audit done.
Checkstyle ends with 1 errors.

```

**Comment by n-aryanshi** [comment]
i want to work on this issue

**Comment by nrmancuso** [comment]
@n-aryanshi issue is not approved, also I would recommend new contributors to avoid indentation issues

**Comment by n-aryanshi** [comment]
okay, I will keep in mind. please tell me on thing do we have to look after issues only which are labelled by "approved"

**Comment by romani** [comment]
yes.
